lib/modules/filteReddit.js

```diff
@@ -494,25 +489,31 @@ function executeCustomFilters(thing) {
 }
 
 const regexRegex = /^\/(.*)\/([gim]+)?$/;
-
 const getStringMatchFilters = _.memoize(type => {
 	const sources = module.options[type].value;
+	const filterMap = {
```

Comment:
do you mean adding the filter type (postTitle, linkFlair, etc.) to the option itself like this?

```javascript
keywords: {
    type: 'table',
    addRowText: '+add filter',
    filter: 'postTitle',... // like this for example
```